app/build.gradle.kts

```diff
@@ -44,33 +41,14 @@ android {
     }
 
     composeOptions {
-        kotlinCompilerExtensionVersion = Versions.Compose.compose
+        kotlinCompilerExtensionVersion = libs.versions.compose.get()
     }
 }
 
 dependencies {
     implementation(project(":shared"))
-    implementation(Deps.material)
-    coreLibraryDesugaring(Deps.desugarJdkLibs)
-    implementation(Deps.AndroidX.core_ktx)
-    implementation(Deps.Ktor.androidCore)
-    implementation(Deps.SqlDelight.runtimeJdk)
-    implementation(Deps.SqlDelight.driverAndroid)
-    implementation(Deps.Coroutines.common)
-    implementation(Deps.Coroutines.android)
-    implementation(Deps.multiplatformSettings)
-    implementation(Deps.koinCore)
-    implementation(Deps.koinAndroid)
-    implementation(Deps.AndroidX.lifecycle_runtime)
-    implementation(Deps.AndroidX.lifecycle_viewmodel)
-    implementation(Deps.AndroidX.lifecycle_viewmodel_extensions)
-
-    implementation(Deps.Compose.activityCompose)
-    implementation(Deps.Compose.ui)
-    implementation(Deps.Compose.uiTooling)
-    implementation(Deps.Compose.foundation)
-    implementation(Deps.Compose.material)
-    implementation(Deps.Compose.Accompanist.swipeRefresh)
-
-    testImplementation(Deps.junit)
+    implementation(libs.bundles.app.ui)
```

Comment:
It took me a moment to realize how much was happening in this line. Bundles seem neat.